src/wstore/models.py

```diff
@@ -92,17 +92,11 @@ class UserProfile(models.Model):
     current_organization = models.ForeignKey(Organization)
```

Comment:
Since you have included directly the current roles as an object, you should remove the organizations field (not longer needed)